CHANGELOG.md

```diff
@@ -1,12 +1,82 @@
 # Bcoin Release Notes & Changelog
 
-## v1.0.x
+## v2.0.0
 
 ### Wallet API changes
 
-Creating a watch-only wallet now requires an `account-key` (or `accountKey`)
-argument. This is to prevent bcoin from generating keys and addresses the user
-can not spend from.
+#### HTTP
+
+- `PUT /wallet/:id` Creating a watch-only wallet now requires an `accountKey`
+argument. This is to prevent bcoin from generating keys and addresses the
+user can not spend from.
+- `POST /wallet/:id/create` Now has a `sign` argument for optional signing
+of transactions.
+
+#### RPC
+
+- Bug fix addresses for the `getnewaddress` command with various networks.
+
+### Network changes
+
+- Regtest have been updated to correspond with other bitcoin
+  implementations, for testing across implementations. Key prefixes have
+  been updated to correspond and segwit is always active. See commit
```

Comment:
"to correspond with testnet"?

Reply:
to correspond with regtest on `bitcoind` and `btcd`

Reply:
should this be reworded?

Reply:
I actually thought words were accidentally missing. I think it would be helpful to explain that testnet addresses are regtest compatible now

Reply:
Reworded